pkg/util/merge/merge_ephemeral_container.go

```diff
@@ -0,0 +1,104 @@
+package merge
```

Comment:
everything here is identical to the `Container` merging function, but acts on `EphemeralContainers` instead.